src/v/raft/persisted_stm.h

```diff
@@ -223,6 +223,10 @@ class persisted_stm_base
 
     const prefix_logger& log() const { return _log; }
 
+    model::offset last_snapshotted_offset() const override {
```

Comment:
can we rename this method to indicate this is the offset of local but not raft snapshot ?